myhpi/core/markdown/extensions.py

```diff
@@ -97,6 +97,24 @@ def quorumify(self, match):
         )
 
 
+class ResolutionPreprocessor(MinutesBasePreprocessor):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.patterns = [
+            (r"\|resolution\|\(\d+([.,]\d{2})?\)\(\S+\)\(\S\)\[(\d+)\|(\d+)\|(\d+)\]", self.resolutify),
```

Comment:
in my local testing, this regex did not match the string that the button inserted into the editor (and me filling in values)